src/applications/vaos/components/FacilityPhone.jsx

```diff
@@ -8,5 +8,14 @@ export default function FacilityPhone({ contact }) {
 
   const [number, extension] = contact.split('x');
 
-  return <Telephone contact={number} extension={extension} />;
+  return (
+    <>
+      <Telephone contact={number} extension={extension} />
+      <br />
+      <h4 className="vads-u-font-family--sans vads-u-display--inline vads-u-font-size--base">
```

Comment:
This seems like it will cause us to skip heading levels when this component is used with `FacilityAddress`, since the "main phone" label isn't always an h3 (you can see this on the homepage refresh details pages). It also seems odd to me that this component contains the "TTY" label, but not the "Main phone" label that's always used with it.

Could we refactor this component to address those concerns? I think you could do something similar to FacilityAddress, where the heading is configurable, and also move the "Main phone" label into that component.